mflowgen/core/run.py

```diff
@@ -202,6 +217,16 @@ def launch_run( s, design, update, backend ):
 
     list_target   = backend + " list"
     status_target = backend + " status"
+    
+    # For all subgraphs in this graph, create step dir and configure them to run
+    for subgraph_step_name in g.all_subgraphs():
+      subgraph_step = g.get_step( subgraph_step_name )
+      subgraph_build_dir = b.get_build_dir( subgraph_step_name )
+      if not os.path.exists( subgraph_build_dir ):
+        os.mkdir( subgraph_build_dir )
+      os.chdir( subgraph_build_dir )
+      s.launch_run( subgraph_step.get_dir(), False, True, backend )
+      os.chdir( '..' )
```

Comment:
Do we still need to generate all the subgraphs dirs after the recursive support is in?

Reply:
Now that we're not planning on showing all levels of hierarchy with `make list`, `make status`, etc., this isn't necessary. I'll remove it

Reply:
Done.. this also fixes the excessive printing issue